staging/src/k8s.io/code-generator/cmd/import-boss/main.go

```diff
@@ -14,49 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// import-boss enforces import restrictions in a given repository.
```

Comment:
I'd either leave this, or at least keep the top line (so that something shows up in godoc).